omod-common/pom.xml

```diff
@@ -33,7 +33,7 @@
 		<dependency>
 			<groupId>io.swagger</groupId>
 			<artifactId>swagger-core</artifactId>
-			<version>1.5.13</version>
+			<version>1.6.2</version>
```

Comment:
LGTM   cc @dkayiwa

Reply:
@achilep   did you these changes with the reff app ??